torch/distributed/fsdp/_common_utils.py

```diff
@@ -61,6 +61,24 @@ def _all_handles(state: _FSDPState) -> List:
     )
 
 
+@no_type_check
+def _module_handles(module: nn.Module, state: _FSDPState) -> List:
```

Comment:
nit: I am also wondering how you feel about argument ordering. I recall that every function/method I refactored had `state: _FSDPState` as the first argument. I do not think this makes any difference functionally, but maybe we keep that convention?